student-cvs/Sheila_FG-7067/index.html

```diff
@@ -0,0 +1,98 @@
+                        to know the terminal and basic commands such as:</p>
+                </section>
+                <section>
+                    <table class="tabla1">
+                        <tr>
+                            <th>Command</th>
+                            <th>Description</th>
+                        </tr>
+                        <tr>
+                            <td>ls</td>
+                            <td>List directory contents: List of files in current folder</td>
+                        </tr>
+                        <tr>
+                            <td>cd</td>
+                            <td>Change current directory: Allows to navigate between folders</td>
+                        </tr>
+                        <tr>
+                            <td>pwd</td>
+                            <td>Print current working directory: Shows the full path of the current directory</td>
+                        </tr>
+                        <tr>
+                            <td>touch</td>
+                            <td>Create a new file given a path and/or name</td>
+                        </tr>
+                        <tr>
+                            <td>cat</td>
+                            <td>Prints the contents of a file given its path</td>
+                        </tr>
+                        <tr>
+                            <td>cp</td>
+                            <td>Allows to copy files given an inicial and end pat</td>
+                        </tr>
+                    </table>
+                </section>
+            </article>
+            <article>
+                <section>
+                    <p class="parrafo">We also come into contact with the alias commands such as:</p>
+                </section>
+                <section>
+                    <table class="tabla2">
+                        <tr>
+                            <th>Alias command</th>
+                            <th>Description</th>
+                        </tr>
+                        <tr>
+                            <td>alias rm="trash"</td>
+                            <td>Allows to delete a file permanently</td>
+                        </tr>
+                        <tr>
+                            <td>alias cl="clear"</td>
+                            <td>Allows to clear the terminal screen</td>
+                        </tr>
+                    </table>
+                    <br>
+                </section>
+            </article>
+            <section>
+                <p class="parrafo">For any inquires, don't hesitate to <a href="contact.html">contact me</a></p>
+            </section>
+            <br>
+            <footer>
+                <div class="link">
+                    <img class="github-mark" src="./Images/github-mark.png" alt="GitHub mark">
+                    <a href="https://github.com/sheifg">Mi perfil de GitHub</a>
```

Comment:
You can use `target="_blank"` for open your GitHub page in a new tab

Reply:
Thanks for the recommendation, I think you are right and it would be better to open it in a new page :)